test/python/circuit/test_parameters.py

```diff
@@ -58,10 +58,10 @@ def test_fix_variable(self):
         qc = QuantumCircuit(qr)
         qc.rx(theta, qr)
         qc.u3(0, theta, 0, qr)
-        qc._parameter_table[theta] = 0.5
+        qc._bind_parameter(theta, 0.5)
         self.assertEqual(qc._parameter_table[theta][0][0].params[0], 0.5)
         self.assertEqual(qc._parameter_table[theta][1][0].params[1], 0.5)
-        qc._parameter_table[theta] = 0.6
+        qc._bind_parameter(theta, 0.6)
```

Comment:
I'd try not to use private methods inside tests. Non blocking.

Reply:
:+1: Fixed in 7fbd82f73a8cfe649c52b946114045d20916d4fe.